flake.lock

```diff
@@ -34,11 +34,11 @@
     },
     "nixpkgs": {
       "locked": {
-        "lastModified": 1704018918,
-        "narHash": "sha256-erjg/HrpC9liEfm7oLqb8GXCqsxaFwIIPqCsknW5aFY=",
+        "lastModified": 1704982786,
+        "narHash": "sha256-K5eJHmL1/kev6WuqyqqbS1cdNnSidIZ3jeqJ7GbrYnQ=",
         "owner": "NixOS",
         "repo": "nixpkgs",
-        "rev": "2c9c58e98243930f8cb70387934daa4bc8b00373",
+        "rev": "a1982c92d8980a0114372973cbdfe0a307f1bdea",
```

Comment:
The channel didn't actually bump yet, but I strongly expect https://hydra.nixos.org/eval/1803537#tabs-inputs to pass soon.

Reply:
Now it did